examples/src/main/java/com/hedera/hashgraph/sdk/examples/ExampleExample.java

```diff
@@ -0,0 +1,62 @@
+package com.hedera.hashgraph.sdk.examples;
+
+import com.hedera.hashgraph.sdk.AccountId;
+import com.hedera.hashgraph.sdk.Client;
+import com.hedera.hashgraph.sdk.PrivateKey;
+import com.hedera.hashgraph.sdk.PublicKey;
+import io.github.cdimascio.dotenv.Dotenv;
+
+import java.util.Objects;
+
+/**
+ * This is a template for examples.
+ *
+ * General description of the example should be here (class-level comment).
+ *
+ * Class access modifier should be default (simplicity and accessibility).
+ */
+class ExampleExample {
```

Comment:
`ExampleExample` is a little bit confusing. I suggest renaming it to `ExampleTemplate`

Reply:
Sounds good to me. Does the template itself make sense?